app/views/users/_data.html.erb

```diff
@@ -44,7 +44,7 @@
       </div>
       <div class="card-body">
         <% if user.edited_courses.any? || user.edited_lectures.any? ||
-              user.given_lectures.any? %>
+              user.given_lectures.any? || user.edited_media.any? %>
```

Comment:
`.exists?` should be a bit faster, see [here](https://stackoverflow.com/a/43774445/9655481)

Reply:
> In Rails 5.1 and up, the performance of any? and exists? has been fixed, so they are now equally performant.
~ from [here](https://www.ombulabs.com/blog/benchmark/performance/rails/present-vs-any-vs-exists.html)

Ah no, I missed that, so `.any?` should also be fine